api.lua

```diff
@@ -44,19 +44,26 @@ require 'passwords'
 
 -- API Endpoints
 -- =============
+-- Wrap all API requests in the same prefix.
+-- TODO: Make /api/v1 be required by removing ().
+-- Do this after Snap! is updated.
+local function api_route(name, path, fn)
```

Comment:
Note to self:

We should replace fn with tbl, and then insert the OPTIONS into that table. Each method should be wrapped in capture_errors automatically.